### ISSUE TITLE
Tests for twisted.names should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7728]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7728|
|Type|enhancement|
|Created|2014-11-29 08:45:09Z|
|Branch|https://github.com/twisted/twisted/tree/names-testcase-names-7728|

This ticket is a follow-up to ticket [ticket:7032].

Attachments:

* [bugfix_7728.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/6de/6dec29b27b448e25fe55e8e0b09d3845f2b286d6/7afdcc667ef0e39ea741bc37c0732b021be1d924.patch) (4416 bytes) - added by eeshangarg on 2014-11-29 09:23:19Z - 
* [bugfix_7728_2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/6de/6dec29b27b448e25fe55e8e0b09d3845f2b286d6/b9e26e97e60012b5fc5a7c57db5665cf421fe3b2.patch) (9138 bytes) - added by eeshangarg on 2015-02-08 07:55:41Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7728 7728
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_names_testcase_names_7728 branches/names-testcase-names-7728
branch_author__ 
status__closed closed
resolution__fixed fixed
component__names names
keywords__easy easy
time__1417250709128273 1417250709128273
changetime__1423591787812981 1423591787812981
version__None None
owner__adiroiban adiroiban
cc__eeshangarg
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_1"></a>|@eeshangarg removed owner|
|-|-|

Hi!

I just added a patch(bugfix_7728.patch​) that changes all of the tests in twisted.names to use the new style of naming test classes as discussed in ticket [ticket:7032].

Thanks & regards,
Eeshan

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

(In [43755]) Branching to names-testcase-names-7728.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @eeshangarg|
|-|-|

Changes looks good. Thanks for helping cleaning up the code.

I have created a branch and waiting for test results.

Since we are touching test names maybe we should also update test method names.

Ex rename testAddressRecord1 to test_addressRecord1 

What do you think? If you agree, please add a new patch on top on names-testcase-names-7728 branch... otherwise I will merge the patch at it is now.


Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_4"></a>|@eeshangarg set owner to @adiroiban|
|-|-|

Thanks for the review, Mr. adiroiban!

I just added a new patch (bugfix_7728_2.patch) created on top of the names-testcase-names-7728 branch, and I changed all of the method names in twisted.names like you advised me to.

If this needs further work, please let me know! :-)

Regards,
Eeshan Garg

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set status to `closed`|
|-|-|

(In [43916]) Merge names-testcase-names-7728: Rename test cases and named in twisted.names.

Author: eeshangarg
Reviewers: adiroiban
Fixes: [#7728](https://github.com/twisted/twisted/issues/7728)

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_7"></a>|@adiroiban commented|
|-|-|

My comment in which I was updating the owner and branch was rejected and it was closed first :(
